student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, parent_permission: true, name: 'Unknown')
+    super(age: age, parent_permission: parent_permission, name: name)
+    @classroom = classroom
+  end
```

Comment:
Well! great job on updating the parameter for the super method. but still need some updates.
Currently, when I try to create a new Student, I get an error of `/person.rb:5:in `initialize': wrong number of arguments`.  I will recommend you follow the previous reviewer's instruction and follow it fully and update the parameter further.
![image](https://github.com/FilbertKimasa/School-library/assets/55840999/ac554cf6-7f3e-475e-a79c-4524f27e03ad)
```suggestion
  def initialize(classroom, age, parent_permission: true, name: 'Unknown')
    super(age, parent_permission: parent_permission, name: name)
    @classroom = classroom
  end
```